src/acvp.c

```diff
@@ -583,6 +586,19 @@ static void acvp_cap_free_kts_ifc_schemes(ACVP_CAPS_LIST *cap_entry) {
     }
     free(cap_entry->cap.kts_ifc_cap->schemes);
 }
+
+static void acvp_cap_free_lms(ACVP_LMS_CAP *cap) {
+    ACVP_LMS_SPECIFIC_LIST *prev = NULL, *curr = cap->specific_list;
```

Comment:
need to check cap <> null or will that always be true ?

Reply:
It is called from a union so it won't be null